### PR TITLE
Update commands and split out getting started

### PR DESCRIPTION
**Pull request by AaronCrawfis**
Update commands and split out getting started

### CODE REVIEW

**Review by orizohar — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/getting-started/quickstarts.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+type: docs
+title: "Try out Dapr quickstarts to learn core concepts"
+linkTitle: "Dapr Quickstarts"
+weight: 50
+description: "Configure Redis for Dapr state management or Pub/Sub"
+---
+
+The [Dapr Quickstarts](https://github.com/dapr/quickstarts) are a collection of tutorials with code samples that are aimed to get you started quickly with Dapr, each highlighting a different Dapr capability.
```
Comment (orizohar) [inline daprdocs/content/en/getting-started/quickstarts.md:9]: Also here link to tagged version

Comment (AaronCrawfis) [reply daprdocs/content/en/getting-started/quickstarts.md]: Done

File: daprdocs/content/en/getting-started/quickstarts.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Quickstarts
+
+| Quickstart               | Description                                                                                                                                                                                    |
+|--------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Hello-world](https://github.com/dapr/quickstarts/tree/master/hello-world)            | Demonstrates how to run Dapr locally. Highlights service invocation and state management.                                                                                                      |
```
[the remark this answers is not in the document]
Comment (AaronCrawfis) [reply daprdocs/content/en/getting-started/quickstarts.md]: Updates

File: daprdocs/content/en/getting-started/install-dapr.md [hunk 3]
```diff
@@ -1,61 +1,11 @@
...
-{{% /codetab %}}
-{{< /tabs >}}
-
-## Install Dapr in self-hosted mode
-
-Running Dapr runtime in self hosted mode enables you to develop Dapr applications in your local development environment and then deploy and run them in other Dapr supported environments.
-
 ### Prerequisites
 
 - Install [Docker Desktop](https://docs.docker.com/install/)
```
Comment (orizohar) [inline daprdocs/content/en/getting-started/install-dapr.md:11]: Also need the Dapr cli for this ...

**Review by orizohar — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/getting-started/install-dapr.md (hunk 3 above)
Comment (AaronCrawfis) [reply daprdocs/content/en/getting-started/install-dapr.md:12]: Added item for cli prereq

File: daprdocs/content/en/getting-started/install-dapr.md [hunk 4]
```diff
@@ -99,7 +54,7 @@ This step will install the latest Dapr Docker containers and setup a developer e
 You can install or upgrade to a specific version of the Dapr runtime using `dapr init --runtime-version`. You can find the list of versions in [Dapr Release](https://github.com/dapr/dapr/releases).
 
 ```bash
-# Install v0.1.0 runtime
+# Install v0.11.0 runtime
 $ dapr init --runtime-version 0.11.0
```
Comment (mukundansundar) [inline daprdocs/content/en/getting-started/install-dapr.md:58]: 0.11.3 is the stable runtime version

File: daprdocs/content/en/getting-started/install-dapr-kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
+---
+type: docs
+title: "How-To: Install Dapr into a Kubernetes cluster"
+linkTitle: "Init Dapr on Kubernetes"
+weight: 30
+description: "Install Dapr in a Kubernetes cluster"
+---
+
+## Install Dapr on a Kubernetes cluster
```
Comment (orizohar) [inline daprdocs/content/en/getting-started/install-dapr-kubernetes.md:9]: Redundant header

File: daprdocs/content/en/getting-started/install-dapr.md (hunk 4 above)
Comment (AaronCrawfis) [reply daprdocs/content/en/getting-started/install-dapr.md]: Updated

File: daprdocs/content/en/getting-started/install-dapr-cli.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
+---
+type: docs
+title: "How-To: Install Dapr CLI"
+linkTitle: "Install Dapr CLI"
+weight: 10
+description: "Install the Dapr CLI to get started with Dapr"
+---
+
+## Dapr CLI installation scripts
```
Comment (orizohar) [inline daprdocs/content/en/getting-started/install-dapr-cli.md:9]: Redundant header

**Review by orizohar — CHANGES_REQUESTED** [review]

**Review by orizohar — APPROVED** [review]

**Pull request merged**